src/blockchain/schema.rs

```diff
@@ -133,26 +132,26 @@ impl<T: ObjectAccess> BtcAnchoringSchema<T> {
         }
     }
 
-    /// Returns the proposal of next anchoring transaction for the given anchoring state.
+    /// Returns the proposal of the next anchoring transaction for the given anchoring state.
     pub fn proposed_anchoring_transaction(
         &self,
         actual_state: &BtcAnchoringState,
     ) -> Option<Result<(Transaction, Vec<Transaction>), BuilderError>> {
         let config = actual_state.actual_configuration();
         let unspent_anchoring_transaction = self.anchoring_transactions_chain().last();
-        let unspent_funding_transaction = self.unspent_funding_transaction();
+        let unspent_funding_transaction = self.unspent_funding_transaction_entry().get();
 
         let mut builder = BtcAnchoringTransactionBuilder::new(&config.redeem_script());
         // First anchoring transaction doesn't have previous.
         if let Some(tx) = unspent_anchoring_transaction {
             let tx_id = tx.id();
 
-            // Checks that latest anchoring transaction isn't a transition.
+            // Check that latest anchoring transaction isn't a transition.
             if actual_state.is_transition() {
                 let current_script_pubkey = &tx.0.output[0].script_pubkey;
                 let outgoing_script_pubkey = &actual_state.script_pubkey();
                 if current_script_pubkey == outgoing_script_pubkey {
-                    trace!("Awaiting for new configuration to become actual.");
+                    trace!("Awaiting for new configuration to become an actual.");
```

Comment:
This sentence seems grammatically incorrect, however I'm not sure what's the right way to fix it.